unit-tests/run-unit-tests.py

```diff
@@ -444,7 +446,12 @@ def test_wrapper( test, configuration=None, repetition=1, sns=None ):
         if test_wrapper_( test, configuration, repetition, retry, sns ):
             return True
         log._n_errors -= 1
-        time.sleep( 1 )  # small pause between tries
+        if no_reset:
+            time.sleep(1)  # small pause between tries
```

Comment:
This should happen only if retry < test.config.retries.
Anyway, same comment as before: move to before line 446 and check if retry > 0. And the log (line 452) should now be taken care of by test_wrapper_(), right?

Reply:
There is a difference between retry and repeat, repeat is currently handles inside, retry outside.
I will think about it.